TS00-Brief-Tabular-Time-Series-In-R.Rmd

```diff
@@ -75,31 +69,22 @@ These data are proxy data for what will be available for 30 years
 on the [NEON data portal](http://data.neoninc.org/ "NEON data")
 for both Harvard Forest and other field sites located across the United States.
 
-####Setting the Working Directory
-The code in this lesson assumes that you have set your working directory to the
-location of the unzipped file of data downloaded above.  If you would like a
-refresher on setting the working directory, please view the
-[Setting A Working Directory In R]({{site.baseurl}}/R/Set-Working-Directory "R Working Directory Lesson")
+**Set Working Directory:** This lessons assumes that you have set your working 
+directory to the location of the downloaded and unzipped data subset. [An overview
+of setting the working directory in `R` can be found here.]({{site.baseurl}}/R/Set-Working-Directory "R Working Directory Lesson") 
 lesson prior to beginning this lesson.
```

Comment:
it looks like i need to revisit grey boxes throughout. Given we will overhaul next week - why don't we just make an issue for this and implement it all at once next week.

I didn't get to the include files in the time series lessons - i just realized that as well. But i also noticed that the include files contain text which could be challenging if we want a few individual buttons. Let's chat next week about all things grey box!